didntreally.CodeReviews.Console.Calculator/Calculator/Program.cs

```diff
@@ -0,0 +1,179 @@
+﻿using System;
+namespace Calculator
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            // Declare variables and then initialize to zero.
+            string num1 = "";
```

Comment:
if you're using VS 2022 you'll notice that some of your variables are greyed out. I suggest you almost always take their advice as they usually make you write less code.

![image](https://user-images.githubusercontent.com/108711476/210889136-2eacdc0f-6df3-4914-9237-2b18df8b7d1a.png)